sfbrigade/datasci-earthquake
# Initialized alembic and migrated

# Description

I have added alembic to the stack and configured it to run in Docker.

#23

## Type of changes

## Testing
- [x] think tests are unnecessary

## How to test
Testing will be another issue and begins with the documentation of `alembic-verify`: https://alembic-verify.readthedocs.io/en/latest/.
